test/fixup-svg-string.js

```diff
@@ -142,3 +142,17 @@ test('fixupSvgString shouldn\'t correct non-image tags', t => {
     t.notEqual(dontFix.indexOf('img/png'), -1);
     t.end();
 });
+
+test('fixupSvgString SHOULD correct hrefs that are NOT data links', t => {
```

Comment:
Maybe also make some tests that use whitespace within tags to try to obfuscate them